doc/sphinx/proof-engine/vernacular-commands.rst

```diff
@@ -608,11 +608,11 @@ file is a particular case of module called *library file*.
 
       This loads and declares the module :n:`@qualid`
       and its dependencies then imports the contents of :n:`@qualid` as described
-      :ref:`here <import_qualid>`. It does not import the modules on which
-      qualid depends unless these modules were themselves required in module
+      for :cmd`Import`. It does not import the modules that
```

Comment:
```suggestion
      for :cmd:`Import`. It does not import the modules that
```

I keep spotting errors like this BTW. Sometimes, you would be better off just applying my suggestion from the GitHub interface (you can apply a batch of suggestions if you switch to the "Changed files" tab).

Reply:
Yeah, hard to get every detail right.  Attention flags when there are so many details at differing levels and the editing sessions are so long (8, 10 and 6 hours each over the last 3 days).

If I accept the suggestion, then I have to pull the new/updated commit from the server.  If I've also made local changes, those need to be shelved before the pull, then somehow merged OR if committed, ???.  My routine has been to go through the changes in the order they appear in the conversation.  I probably would have to make two passes through the conversation: first, to get the suggestions, then second for other changes.

So, um, maybe.